xlators/cluster/afr/src/afr-self-heal-entry.c

```diff
@@ -230,12 +234,25 @@ afr_new_entry_mark_status(call_frame_t *frame, loc_t *loc,
             goto lookup;
         }
     }
+    if (iatt->ia_type == IA_IFLNK && dst_hardlink) {
```

Comment:
If this conditions holds true, I think we can directly goto lookup from here too, as the steps look similar.

Reply:
@xhernandez  Can you please check once and merge if you don't have any concerns?

Reply:
It seems ok, but I'm not very familiarized with all the details of AFR's self-heal code. It would be better that @pranithk reviewed it.